Packs/Devo/Integrations/Devo_v2/Devo_v2_test.py

```diff
@@ -371,25 +412,17 @@ def test_run_query(mock_query_results, mock_args_results):
     assert (results[1]["HumanReadable"]).find("Devo Direct Link") != -1
     assert len(results) == 2
     assert results[0]["Contents"][0]["context"] == "CPU_Usage_Alert"
-    expected_columns = ','.join(field.strip() for field in MOCK_QUERY_ARGS['fields_to_view'].split(','))
-    result = results[0]["Contents"][0]
-    assert all(column in result for column in expected_columns.split(',')), (
-        f"Not all columns present in the dictionary. Missing columns: "
-        f"{', '.join(column for column in expected_columns.split(',') if column not in result)}"
-    )
 
 
 @patch("Devo_v2.READER_ENDPOINT", MOCK_READER_ENDPOINT, create=True)
 @patch("Devo_v2.READER_OAUTH_TOKEN", MOCK_READER_OAUTH_TOKEN, create=True)
 @patch("Devo_v2.demisto.args")
 @patch("Devo_v2.ds.Reader.query")
-def test_run_query_with_empty_fields_to_view_param(mock_query_results, mock_args_results):
+def test_run_query_with_invalid_column_name(mock_query_results, mock_args_results):
     mock_query_results.return_value = copy.deepcopy(MOCK_QUERY_RESULTS)
-    mock_args_results.return_value = MOCK_QUERY_ARGS_EMPTY_FIELDS_TO_VIEW_PRAM
-    try:
+    mock_args_results.return_value = MOCK_QUERY_ARGS_INVALIDE_COLUMN_NAME
+    with pytest.raises(ValueError, match=re.escape("Fields ['abcd'] not found in query result")):
         run_query_command(OFFSET, ITEMS_PER_PAGE)
-    except ValueError as e:
-        assert "fields_to_view cannot be an empty string!" in str(e), f"Unexpected exception message: {e}"
```

Comment:
**question(blocking):** Does this test run? I don't see this error thrown in the code anywhere.

Reply:
My bad; I just found it.